creature/LaserLlama; Laser Llama.json

```diff
@@ -0,0 +1,242 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "LL Laser Llama",
```

Comment:
\>spaces in `json`

![bad](https://github.com/user-attachments/assets/e616930d-465a-4f2c-9692-370ee1c88563)

Reply:
Yeah valid